### ISSUE TITLE
Many issues in Firefox

### ISSUE THREAD

**Issue by vincentbernat** [ericduran/chromeHAR#9]
Hi!

ChromeHAR exhibits many problems in Firefox. The layout is mangled and some elements are missing. See the following screenshot:

![screenshot-2013-04-20--14:31:40](https://f.cloud.github.com/assets/631446/405114/4b9fc0e2-a9b6-11e2-90ee-a075692ab72b.jpg)

**Comment by ericduran** [comment]
Yep, this is because the webkit code only uses the -webkit prefix.

This is what happens when you take code from webkit ;-) Ha

I'll fix all the prefix.

**Comment by cvan** [comment]
I ran the CSS through a script that added `-moz-` prefixes wherever there were `-webkit-` ones:

# Before

![](http://f.cl.ly/items/3k3L1q3S153W211o101y/Screen%20Shot%202014-01-23%20at%2012.17.22%20AM.png)

# After

![](http://f.cl.ly/items/0v2I3P3G1Z162m2j3v0l/Screen%20Shot%202014-01-23%20at%2012.17.05%20AM.png)

LOL not much of a difference. Any ideas?

**Comment by ericduran** [comment]
So I just pushed a commit to fix some of this.

It looks a little better but not perfect. 

![](https://dl.dropboxusercontent.com/u/16395906/chromehar-firefox.png)

I'll continue making fixes for this. FYI: I added a generic.css file where I'm overwriting the webkit specific stuff by hand.

Sadly it's a bit difficult to script since chrome/webkit is using non w3c standard features (ex. webkit-mask-image)
